source/features/pr-jump-to-first-non-viewed-file.tsx

```diff
@@ -28,5 +28,8 @@ void features.add({
 	include: [
 		pageDetect.isPRFiles
 	],
+	exclude: [
+		() => document.title.startsWith('Commit range not found · Pull Request')
+	],
```

Comment:
Can you move this to `github-url-detection`?

Reply:
`isPr404`?

Reply:
`isPR404Commit`